### PR TITLE
fix: closing addblock drawer action

### PR DESCRIPTION
**Pull request by DanChae03**
# Description

### Issue

Closing the Addblock drawer used to switch the screen to the journeyflow view. This has now been fixed, and the screen will not switch, and the properties drawer will become visible. 

Also, closing the properties drawer has been fixed to redirect to the journeyflow view ONLY on desktop - closing it on mobile/ipad (smaller than medium breakpoint) now redirects to the content view.

[Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/38213939/todos/7586174105)

### Solution

_Provide a detailed description of how exactly this task will be accomplished. This can be something technical. What specific steps will be taken to achieve the goal? This should include details on service integration, job logic, implementation, etc._

# External Changes

_If you have made changes to external services, need to add additional values to Doppler, or need to add something new to the database, explain it here. This may include updates to third-party services, changes to infrastructure configuration, integration with external APIs, etc._

# Additional information

_Provide any additional information that might be useful to the reviewer in evaluating this pull request. This could include performance considerations,design choices, etc._

### CODE REVIEW

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/AddBlock/AddBlock.spec.tsx [hunk 1]
```diff
@@ -130,14 +135,13 @@ describe('AddBlock', () => {
       </MockedProvider>
     )
 
-    expect(screen.getByText('activeSlide: 1')).toBeInTheDocument()
```
Comment (levi-zustiak) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/AddBlock/AddBlock.spec.tsx:133]: Could I have you add an assertion that `activeCanvasDetailsDrawer` is set to `AddBlock` similarly to the previous version?

File: apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/AddBlock/AddBlock.spec.tsx [hunk 2]
```diff
@@ -130,14 +135,13 @@ describe('AddBlock', () => {
...
     await waitFor(() =>
```
Comment (levi-zustiak) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/AddBlock/AddBlock.spec.tsx:138]: There's aren't necessarily your changes, but if we can also clean some of the extra stuff that's not needed while we're here.

```suggestion
    const closeButton = screen.getByTestId('X2Icon')
    
    expect(closeButton).toBeInTheDocument()
    
    await waitFor(() => userEvent.click(closeButton))
```

**Review by levi-zustiak — CHANGES_REQUESTED** [review]
Just a couple small comments, but looks good man! Good stuff

**Review by levi-zustiak — APPROVED** [review]
Looks good 👍

**Review by csiyang — APPROVED** [review]

**Pull request merged**